addon/adapters/rest.js

```diff
@@ -4,12 +4,16 @@
   @module ember-data
 */
 
-import $ from 'jquery';
+import Ember from 'ember';
+import fetch, { Response } from 'fetch';
+//FIXME: We need to add a direct export of `serializeQueryParams`
+import { serializeQueryParams } from 'ember-fetch/mixins/adapter-fetch';
```

Comment:
I've been wondering where the best place for this to live is. In a jQuery-less world, `$.param()` is gone and Ember should have some replacement.

Reply:
I feel like `ember-fetch` is good place for now. I am planing on doing a PR to move it out of `ember-fetch/mixins/adapter-fetch`. But a broader discussion is needed about `fetch` usage: `fetch` on it own is not a suitable replacement for `$.ajax` in my opinion. `fetch` is too low level.

Reply:
Another option might be to use `URLSearchParams` provided by `ember-url` (with polyfills). It is a standard, but might be less convenient then a `$.param()`.

Reply:
what's better? `URLSearchParams` from `ember-url`, or `serializerQueryParams` from the fetch adapter?

Reply:
Using `URLSearchParams` is not an option after investigation – it won't serialise nested params. I think wee should expose something like `import { param } from 'ember-fetch';`

Reply:
https://github.com/ember-cli/ember-fetch/pull/130

Reply:
@tchak ember-fetch has been updated!